NwPluginAPI/Core/Respawn.cs

```diff
@@ -4,32 +4,62 @@ namespace PluginAPI.Core
 	using static Respawning.RespawnEffectsController;
 
 	/// <summary>
-	/// Handles respawning
+	/// Handles respawning.
 	/// </summary>
 	public static class Respawn
 	{
 		/// <summary>
-		/// Gets the amount of NTF tickets left.
+		/// Gets or sets the amount of NTF tickets left.
 		/// </summary>
-		public static float NtfTickets => RespawnTokensManager.GetTeamDominance(SpawnableTeamType.NineTailedFox);
+		public static float NtfTickets
+		{
+			get => RespawnTokensManager.GetTeamDominance(SpawnableTeamType.NineTailedFox);
+			set => AddTickets(SpawnableTeamType.NineTailedFox, value);
```

Comment:
This will not "set" the number of tickets but it will add "value" on the current number of tickets.